### DISCUSSION TITLE
How do I build the spec?

### DISCUSSION THREAD

**Discussion by JustAnotherCodemonkey** [gpuweb/gpuweb#4198] [Q&A] [answered]
Running `make` returns an error (`LINK ERROR: Multiple possible 'TypeError' idl refs.`). What's the proper way to build? I can't find any documentation on this.

Running macos 13.4 intel chip. I installed Bikeshed via pip.

**Comment by kainino0x** [comment]
This appears to be a new build regression in a recent bikeshed release. I'll convert this into an issue and take a look.

**Reply by JustAnotherCodemonkey** [reply]
But `make` is the correct command, yes?

**Reply by kainino0x** [reply, marked as answer]
Yes. The fact that you ran into this error means your build toolchain seems to be set up correctly, we just need to fix the build for the latest Bikeshed release.

**Reply by kainino0x** [reply]
Realized github didn't clearly auto link to the issue I filed. It's https://github.com/gpuweb/gpuweb/issues/4199

**Reply by kainino0x** [reply]
Should be fixed now!
